addon-info.json

```diff
@@ -1,10 +1,10 @@
 {
   "name" : "ensime-vim",
-  "version" : "0.1",
-  "author" : "Olivier 'yazgoo' Abdesselam",
-  "maintainer" : "Olivier 'yazgoo' Abdesselam",
+  "version" : "0.2",
+  "author" : "Olivier 'yazgoo' Abdesselam, Jorge Vicente Cantero (jvican)",
```

Comment:
I'ld be enclined to remove all these author fields, or at least point them to the ensime-vim project, and create an AUTHOR.md, would that be fine with you ?